### PR TITLE
FUNCTIONS WITH PARAMETERS EXTRA

### PR DESCRIPTION
**Pull request by leonel-acosta**

### CODE REVIEW

File: js-functions/js-functions_functions-with-parameters-extra/js/index.js [hunk 1]
```diff
@@ -6,7 +6,10 @@ console.clear();
    The formula is: square = number * number
 */
 
-function printSquare(number) {}
+function printSquare(number) {
+   square = number * number
```
Comment (fadiktekin) [inline js-functions/js-functions_functions-with-parameters-extra/js/index.js:10]: declare with `const`
`const square = number * number`

File: js-functions/js-functions_functions-with-parameters-extra/js/index.js [hunk 2]
```diff
@@ -19,8 +22,15 @@ printSquare(5);
 
 // Uncomment the following function calls and implement the function printCircumference
 
-// printCircumference(4);
-// printCircumference(6);
+let radius = 39
+
+function printCircumference(radius){
+   circumference = 2 * Math.PI * radius
```
Comment (fadiktekin) [inline js-functions/js-functions_functions-with-parameters-extra/js/index.js:28]: `const circumference`

File: js-functions/js-functions_functions-with-parameters-extra/js/index.js [hunk 3]
```diff
@@ -32,5 +42,13 @@ printSquare(5);
 
 // Uncomment the following function calls and implement the function printRectangleArea
 
-// printRectangleArea(5, 7);
-// printRectangleArea(3, 4);
+let width = 220;
+let length = 440;
+
+function printRectangleArea(width, length){
+   area = width * length
```
Comment (fadiktekin) [inline js-functions/js-functions_functions-with-parameters-extra/js/index.js:49]: `const area`

Comment (leonel-acosta) [reply js-functions/js-functions_functions-with-parameters-extra/js/index.js]: fixed

File: js-functions/js-functions_functions-with-parameters-extra/js/index.js (hunk 1 above)
Comment (leonel-acosta) [reply js-functions/js-functions_functions-with-parameters-extra/js/index.js]: fixed

**Pull request merged**